change/@ni-nimble-components-3680232d-ba1a-47f0-8cdc-dc6effd176ed.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We have pretty thorough interactive column size tests. Can you work with @atmgrifter00 to buddy and decide appropriate test coverage? Will move back to draft since requesting a buddy review

Reply:
I think it would be sufficient to update this [parameterized test](https://github.com/ni/nimble/blob/29b790b9e5ee15182ee9efed772abef3e3c9b15f/packages/nimble-components/src/table/tests/table-column-sizing.spec.ts#L531) to essentially run the same tests but to use a touch event instead. This would require just updating this page object method to take another argument to determine which type of event to use (and then double the set of tests).

Reply:
I started to update the tests, but then realized it would be cleaner if we used pointer events (which are meant to unify handling of mouse, touch, and pen inputs) instead of mouse and touch events. Then we don't need additional test cases, as long as we update the page object to send pointer events instead of mouse/touch events.

Reply:
Ahh yea, I remember going down that path and looks like we captured an issue for switching to pointer events: https://github.com/ni/nimble/issues/1362

May need to double check if the issues were ran into with Firefox are still a problem, it's been a while

Reply:
The [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1556240) mentioned in that issue is still open, but testing locally, I don't see any weird behavior like drag-release selecting rows. I think we can close our issue with this change.

Reply:
I'd be curious if with this change the `glass-overlay` div can be removed (and the styling related to it) from the table template. It's existence was at least [called out](https://github.com/ni/nimble/issues/1362#issue-1812224551) in the issue for investigating pointer events as something that pointer events could remove the need for.

Reply:
Once I started calling `setPointerCapture`, the `glass-overlay` div became unnecessary, so I removed it.

Reply:
While the changes seem fine, obviously the failing tests are a concern.

Reply:
Using `setPointerCapture` required some changes to the page object. I had missed running those locally, because they were under a second top-level `describe`.